src/fswAlgorithms/pointCloudProcessing/initializeICP/_UnitTest/test_initICP.py

```diff
@@ -0,0 +1,183 @@
+# ISC License
+#
+# Copyright (c) 2023, Laboratory for Atmospheric Space Physics, University of Colorado Boulder
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+
+import glob
+import inspect
+import numpy as np
+import os
+import pickle
+import pytest
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
+path = os.path.dirname(os.path.abspath(filename))
+bskName = 'Basilisk'
+splitPath = path.split(bskName)
+
+from Basilisk.utilities import RigidBodyKinematics as rbk
+from Basilisk.utilities import SimulationBaseClass, macros
+
+from Basilisk.fswAlgorithms import initializeICP
+from Basilisk.architecture import messaging
+
+
+def test_all_valid():
+    """
+    Unit test with valid ICP message and valid point cloud
+    """
+    icp_init_sim(validICP = True, validCloud = True, normalize = True)
+
+def test_all_valid():
```

Comment:
:large_orange_diamond: Copy and paste? Function name could be `test_all_valid_no_normalization` or something.

Reply:
Done!